src/frontend/src/components/SnackBar/SnackBar.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
+import { makeStyles } from '@material-ui/core/styles';
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    '& > * + *': {
+      marginTop: theme.spacing(2),
+    },
+  },
+}));
+
+export default function CustomizedSnackbar(props) {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+  const { newPosts } = props;
+
+  useEffect(() => {
+    setOpen(true);
+  }, []);
+
+  const handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setOpen(false);
+  };
+
+  return (
+    <div className={classes.root}>
+      <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
+        <Alert onClose={handleClose} severity="info">
+          There are {newPosts} new post(s) available!
```

Comment:
I would just say `There are new posts available!` so you don't have to deal with the `is/are` and `post/posts` differences.  Otherwise, you'd have to make this an `if` check on the length of `newPosts` and do one thing for 1, and something else for the other cases.

Reply:
Fixed, I was just using it for testing purposes